apps/meteor/app/livechat/server/lib/LivechatTyped.ts

```diff
@@ -1000,7 +1000,11 @@ class LivechatClass {
 			throw new Error('department-not-found');
 		}
 
-		await Promise.all([LivechatDepartmentAgents.disableAgentsByDepartmentId(_id), LivechatDepartment.archiveDepartment(_id)]);
+		await Promise.all([
+			LivechatDepartmentAgents.disableAgentsByDepartmentId(_id),
+			LivechatDepartment.archiveDepartment(_id),
+			notifyOnLivechatDepartmentAgentChangedByDepartmentId(_id),
```

Comment:
should this be outside? i notice everywhere you're leaving to run async except here